CHANGELOG.md

```diff
@@ -7,9 +7,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Breaking changes
 
+- Drop support for Rails < 6.1
```

Comment:
I'd append again "This also drops support for MySQL, which had only been present for Rails < 6."